vsts/pipelines/copySdksFromProdToStorageAccount.yml

```diff
@@ -37,9 +37,10 @@ jobs:
           DEV_STORAGE_SAS_TOKEN: $(DEV-STORAGE-SAS-TOKEN)
           SANDBOX_STORAGE_SAS_TOKEN: $(SANDBOX-STORAGE-SAS-TOKEN)
           PROD_STORAGE_SAS_TOKEN: $(PROD-STORAGE-SAS-TOKEN)
+          PERSONAL_STORAGE_SAS_TOKEN: $(${{ parameters.destinationStorageAccountName }}-PERSONAL-STORAGE-SAS-TOKEN)
```

Comment:
nit: I'd be in favor of making sure the entire variable is capitalized independent of what we provide to the pipeline by using the [`upper` function](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#upper):

```
$(${{ upper(parameters.destinationStorageAccountName) }}-PERSONAL-STORAGE-SAS-TOKEN )
```

I'd make the same comment in all instances of `PERSONAL_STORAGE_SAS_TOKEN` in this PR.